projects/client/Unit/Unit.csproj

```diff
@@ -22,9 +22,8 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.2.0" />
     <PackageReference Include="NUnit" Version="3.12.0" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.15.1" />
-    <!--<PackageReference Include="PublicApiGenerator" Version="8.1.0" />-->
     <PackageReference Include="Mono.Cecil" Version="0.10.3" />
-
+    <PackageReference Include="PublicApiGenerator" Version="8.1.0" />
```

Comment:
8.1.0 had a bug where it didn't sort things consistently between platforms, so that's why it was commented out, and the source was manually brought it with the fix. 

Since then, newer versions have come out, so it should be fine to reference the latest version, 10.0,2, and delete the manually included source.

I've been meaning to open a PR to do this on `master` but haven't had a chance to yet.

Reply:
Ah gotcha, I was wondering how it worked on `master`! I'll give that an attempt.